client/lib/wp/browser.js

```diff
@@ -11,15 +12,20 @@ import wpcomUndocumented from 'lib/wpcom-undocumented';
 import config from 'config';
 import wpcomSupport from 'lib/wp/support';
 
+const addSyncHandlerWrapper = config.isEnabled( 'sync-handler' );
 let wpcom;
 
 if ( config.isEnabled( 'oauth' ) ) {
-	const oauthToken = require( 'lib/oauth-token' ),
-		requestHandler = require( 'lib/wpcom-xhr-wrapper' );
+	const oauthToken = require( 'lib/oauth-token' );
+	const requestHandler = addSyncHandlerWrapper
```

Comment:
It isn't duplication: the requires are different.

Not sure if it's correct, though. The first block has `lib/wpcom-xhr-wrapper` both times. The second block has `wpcom-proxy-request` and then `lib/wpcom-xhr-wrapper`.